kairon/shared/pos/processor.py

```
@@ -866,5 +942,147 @@ def reject_pos_order(self, session_id: str, order_id: int) -> Dict[str, Any]:
+
+        return data[0]["res_id"]
+
+    def create_user(
+            self,
+            session_id: str,
+            bot: str,
+            client_name: str,
+            login: str,
+            password: str,
+            name: str,
+            partner_id: int = None,
```

Ruff (0.15.15)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)